<launch>


	<!-- f a c e -->
	<node name="bender_face" pkg="bender_face" type="face.py" output="screen" respawn="true"/>

	<!-- vision  -->
	<include file="$(find bender_vision)/launch/face_recognition.launch"/>
	
	<!-- macro -->
	<node name="search_face" pkg="bender_macros" type="search_face" output="screen">
		<rosparam file="$(find bender_macros)/config/SearchFace.yaml"/>
	</node>
	
</launch>
